integr8ly/tutorial-web-app
# feat(RhmiConfig): add a watch to the rhmiconfig in the settings page

## Motivation

Improve usability by watching for changes in the RHMIConfig CR instead of waiting for the user to refresh the page to view the changes  

## What
Add a watcher to the RHMIConfig CR endpoint

## Why
Improve usability

## How
Polling the RHMIConfig CR endpoint and in case of changes updating the state of the settings page

## Verification Steps

1) Install the webapp
2) Run the webapp against your cluster. [See how to do here](https://github.com/integr8ly/tutorial-web-app#3-cluster-configuration)
3) Go to the settings page by clicking on the cog icon (located in the top right corner of the webapp) 
4) Change the spec block of the `RHMIConfig` CR and verify if the change has appeared in the webapp
 

## Checklist:


## Progress
